scripts/checkInternal.js

```diff
@@ -40,11 +40,9 @@ for (let lineIdx = 0; lineIdx < lines.length; ++lineIdx) {
     continue;
```

Comment:
Could we now change our regex to simply `@internal` (probably don't even need a regex for that)?

Reply:
Simplified